edx_when/models.py

```diff
@@ -22,6 +23,9 @@
     Schedule = None
 
 
+LOG = logging.getLogger(__name__)
```

Comment:
Ooh, I've seen `log` and `logger` but not `LOG` before. Is this your preference or like, where edx is trying to get to?

Reply:
Honestly, I'm inconsistent between `log` and `LOG`, and went w/ the latter in this case because it's a top-level object (although it's not a constant).

Reply:
Really, I should just do the same thing that the rest of the code is doing.